src/commcare_cloud/ansible/roles/pgbouncer/defaults/main.yml

```diff
@@ -1,8 +1,10 @@
 pgbouncer_port: 6432
-pgbouncer_pid_file: "/var/run/postgresql/pgbouncer.pid"
+pgbouncer_pid_file: "/var/run/postgresql/pgbouncer-{{ item }}.pid"
 pgbouncer_config_home: /etc/pgbouncer
-pgbouncer_ini: "{{ pgbouncer_config_home }}/pgbouncer.ini"
-pgbouncer_users: "{{ pgbouncer_config_home }}/userlist.txt"
+pgbouncer_ini: "{{ pgbouncer_config_home }}/pgbouncer-{{ item }}.ini"
+pgbouncer_users: "{{ pgbouncer_config_home }}/userlist-{{ item }}.txt"
```

Comment:
is it necessary to have different files for each process?

Reply:
For the config file, yes, because we need to specify different log files, pid files, and socket directories for each process.
For the userlist, ah! Maybe not?

Reply:
Fixed in the last commit, we now use a single userlist file.